python-checks/src/test/resources/checks/argumentType.py

```diff
@@ -267,3 +267,15 @@ class EnumA(Enum):
         C = 3
 
     len(EnumA)
+
```

Comment:
cause in this example it compares AnotherClass to be compatible with DecoratedClass and since it has member `x` as `DecoratedClass` has - it says that it is compatible type

Reply:
Indeed, thanks